sound/soc/sof/sof-priv.h

```diff
@@ -425,6 +425,8 @@ int snd_sof_dsp_mailbox_init(struct snd_sof_dev *sdev, u32 dspbox,
 			     size_t dspbox_size, u32 hostbox,
 			     size_t hostbox_size);
 int snd_sof_ipc_valid(struct snd_sof_dev *sdev);
+int sof_ipc_tx_msg(struct snd_sof_ipc *ipc, u32 header, void *tx_data,
+		   size_t tx_bytes, void *rx_data, size_t rx_bytes);
 int sof_ipc_tx_message(struct snd_sof_ipc *ipc, u32 header, void *tx_data,
 		       size_t tx_bytes, void *rx_data, size_t rx_bytes);
```

Comment:
We now have functions which are doing basically the same thing, but one is able to send larger messages. I propose to export just one function to avoid confusion for readers.